cobaya/samplers/mcmc/mcmc.py

```diff
@@ -47,6 +47,7 @@ class MCMC(CovmatSampler):
     learn_every: NumberWithUnits
     output_every: NumberWithUnits
     callback_every: NumberWithUnits
+    temperature: float
```

Comment:
Optional[float]

Reply:
Or just have temperature=1 by default, with tests changed to "if temperature!=1", which would be slightly more consistent (currently temperature is applied even if 1, temperature 0 has same effect as None)

Reply:
Done.